benchmark/infrastructure/BenchmarkMeasurementContainer.h

```diff
@@ -329,6 +337,12 @@ class ResultGroup : public BenchmarkMetadataGetter {
     return (*resultEntries_.back());
   }
 
+  /*
+  Delete the given `ResultEntry` from the group. Note: Because the group has
+  ownership of all contained entries, this will delete the `ResultEntry`.
```

Comment:
Do you mean "this will invalidate the argument `entry`"?

Reply:
Yes. Changed in [b0dd92b](https://github.com/ad-freiburg/qlever/pull/1207/commits/b0dd92bb0ab3f87f70ce748812b8582a0b432ee4).